components/next_steps_view/steps.ts

```diff
@@ -2,7 +2,10 @@
 // See LICENSE.txt for license information.
 import {createSelector} from 'reselect';
 
-import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import {
+    makeGetCategory,
+    getCreateGuidedFirstChannelCTATreatment,
```

Comment:
Nit: reduce the name here given it's very wordy, maybe:
```suggestion
    getOnboardingCreateChannelTreatment,
```

Reply:
ok!